protos/in_toto_attestation/predicates/vuln/v1/vuln.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package in_toto_attestation.predicates.vuln.v1;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/in-toto/attestation/go/predicates/vuln/v1";
+option java_package = "io.github.intoto.attestation.predicates.vuln.v1";
+
+// Validation of all fields is left to the users of this proto.
+message Vuln {
+  Scanner scanner = 1;
+  ScanMetadata scan_metadata = 2;
+}
+
+message Scanner {
+  string uri = 1;
+  string version = 2;
+  VulnDatabase database = 3;
+  Result result = 4;
+}
+
+message VulnDatabase {
+  optional string uri = 1;
+  optional string version = 2;
+  google.protobuf.Timestamp last_update = 3;
+}
+
+message Result {
+ repeated Vulnerability vulnerabilities = 3;
+}
+
+message Vulnerability {
+  string id = 1;
+  Severity severity = 2;
```

Comment:
I think this is where we need resolution to https://github.com/in-toto/attestation/issues/342

As defined _here_ a given vulnerability can only have a single severity and method.  In practice there are many ways to measure severity and users may need to express both in the vuln attestation.

We can see this in practice in [the example](https://github.com/in-toto/attestation/blob/main/spec/predicates/vuln.md#:~:text=CVE%2D123%22%2C-,%22severity%22%3A%20%5B,-%7B%20%22method%22).

So perhaps this should be repeated?

Reply:
I'll make all these changes. 👍🏻